diagnostic.md

```diff
@@ -16,22 +17,29 @@ Place your responses inside the fenced code-blocks where indivated by comments.
 
   ```md
     # < Your Response Here >
+    Not quite sure what this is asking for... but many Profile can have many Favorites and a Favorite can also have many Movies.
```

Comment:
We were looking for you to write out a table here, maybe the instructions should be more explicit however